src/Views/Appearance.vala

```diff
@@ -126,13 +149,17 @@ public class PantheonShell.Appearance : Gtk.Grid {
         attach (translucency_switch, 1, 5);
         attach (text_size_label, 0, 6);
         attach (text_size_modebutton, 1, 6, 2);
+        attach (dyslexia_font_label, 0, 7);
+        attach (dyslexia_font_switch, 1, 7);
+        attach (dyslexia_font_description_label, 1, 8);
```

Comment:
This line actually needs to span two columns. It's not obvious on Hera because we don't show the Style and Accent sections.

```suggestion
        attach (dyslexia_font_description_label, 1, 8, 2);
```